apps/monk-test-app/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@monkvision/monk-test-app",
+  "version": "1.0.0",
+  "license": "BSD-3-Clause-Clear",
+  "packageManager": "yarn@3.2.4",
+  "description": "MonkJs test app with react and typescript",
+  "author": "monkvision",
+  "private": true,
+  "dependencies": {
+    "@testing-library/dom": "^8.20.0",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.5.2",
+    "@types/node": "^16.18.18",
+    "@types/react": "^18.0.28",
+    "@types/react-dom": "^18.0.11",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "typescript": "^4.9.5",
+    "web-vitals": "^2.1.4"
+  },
+  "devDependencies": {
+    "@monkvision/eslint-config-base": "1.0.0",
+    "@monkvision/eslint-config-typescript": "1.0.0",
+    "@monkvision/eslint-config-typescript-react": "1.0.0",
+    "@monkvision/prettier-config": "1.0.0",
+    "@monkvision/typescript-config": "1.0.0",
+    "@typescript-eslint/eslint-plugin": "^5.43.0",
+    "@typescript-eslint/parser": "^5.43.0",
+    "eslint": "^8.29.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-formatter-pretty": "^4.1.0",
+    "eslint-plugin-eslint-comments": "^3.2.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jest": "^25.3.0",
+    "eslint-plugin-jsx-a11y": "^6.7.1",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-promise": "^6.1.1",
+    "eslint-plugin-react": "^7.32.2",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-utils": "^3.0.0",
+    "jest": "^29.3.1",
+    "mkdirp": "^1.0.4",
+    "prettier": "^2.7.1",
+    "regexpp": "^3.2.0",
+    "rimraf": "^3.0.2",
+    "ts-jest": "^29.0.3"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --no-watch --no-watchAll",
+    "eject": "react-scripts eject"
```

Comment:
you did not add a lint script ! please add a lint script (same as other projects)

Reply:
Done